bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -261,11 +262,19 @@ pub mod pallet {
 				}
 			})?;
 
-			// Reward relayer from the sovereign account of the destination parachain
-			// Expected to fail if sovereign account has no funds
+			// Reward relayer from the sovereign account of the destination parachain, only if funds
+			// are available
 			let sovereign_account = sibling_sovereign_account::<T>(channel.para_id);
 			let delivery_cost = Self::calculate_delivery_cost(message.encode().len() as u32);
-			T::Token::transfer(&sovereign_account, &who, delivery_cost, Preservation::Preserve)?;
+			let amount = T::Token::reducible_balance(
+				&sovereign_account,
+				Preservation::Preserve,
+				Fortitude::Polite,
+			)
+			.min(delivery_cost);
+			if amount > BalanceOf::<T>::zero() {
```

Comment:
Could be simplified to:
```suggestion
			if !amount.is_zero() {
```

but feel free to ignore, since it'll require another PR in another repo :)